tests/visualization_tests/test_parallel_coordinate.py

```diff
@@ -129,6 +129,67 @@ def test_plot_parallel_coordinate_categorical_params() -> None:
     assert figure.data[0]["dimensions"][2]["ticktext"] == ("net", "una")
 
 
+def test_plot_parallel_coordinate_categorical_numeric_params() -> None:
+    # Test with categorical params that can be interpreted as numeric params.
+    study_categorical_params = create_study()
+    study_categorical_params.add_trial(
+        create_trial(
+            value=0.0,
+            params={"category_a": 2, "category_b": 2},
+            distributions={
+                "category_a": CategoricalDistribution((1, 2, 10)),
+                "category_b": CategoricalDistribution((-1, 1, 2)),
```

Comment:
Could you add another parameter sampled from another distribution except for categorical distribution? This is because the introduced [sort logic](https://github.com/optuna/optuna/pull/2821/files#diff-fb4c806a38d57b2092f804687882f64ad7ad51616bb06d183acdd72acee671d4R191-R193) affects such values.

Reply:
Absolutely right. I implemented a test with sample from multiple distribution. Please check it right.

Reply:
great! thanks!

Reply:
Ah, I think the added test case, `test_plot_parallel_coordinate_sample_from_multiple_distribution` is enough. How about removing this `test_plot_parallel_coordinate_categorical_numeric_params` function and rename `test_plot_parallel_coordinate_sample_from_multiple_distribution` with `test_plot_parallel_coordinate_with_categorical_numeric_params`?

Reply:
Ah, I think the added test case, `test_plot_parallel_coordinate_sample_from_multiple_distribution` is enough. How about removing this `test_plot_parallel_coordinate_categorical_numeric_params` function and rename `test_plot_parallel_coordinate_sample_from_multiple_distribution` with `test_plot_parallel_coordinate_with_categorical_numeric_params`?

Reply:
Thanks your comment! I wonder if we need the case of sort that sort key are  multiple columns. If you need it , I will implement the case to the merged test. What do you think?

Reply:
Yes!! The proposal sounds great. I appreciate you for implementing your additional test cases.

Reply:
Sorry for my late. I merged two test and added the case that proposed previous comment.